File: app/src/main/java/com/fsoon/android/fsfirsttemplate/view/base/BaseActivity.java
package com.fsoon.android.fsfirsttemplate.view.base;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.fsoon.android.fsfirsttemplate.R;

public class BaseActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_base);
    }
}

File: app/src/main/kotlin/com/fsoon/android/fsfirsttemplate/view/base/BaseActivity.kt
package com.fsoon.android.fsfirsttemplate.view.base

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.fsoon.android.fsfirsttemplate.R

open class BaseActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_base)
    }
}